virttest/virt_vm.py

```diff
@@ -659,6 +659,7 @@ def get_address(self, index=0):
             raise VMMACAddressMissingError(index)
         if self.ip_version == "ipv4":
             # Get the IP address from arp cache
+            del self.address_cache[nic.mac.lower()]
```

Comment:
It is not good idea to determine the stale ip from this code path, we would endup in above mentioned issues.